### PR TITLE
{chem,math}[foss/2017b] Cantera v2.3.0, SUNDIALS v2.7.0 + deps

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
node2597.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/9c3c8536b302e1f3732a856258b67a7f for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 8 out of 8 (5 easyconfigs in this PR)
dell.lan - Linux fedora 27, Intel(R) Core(TM) i7-4600U CPU @ 2.10GHz, Python 2.7.14
See https://gist.github.com/ed3b9e8517bd491a6f32772178c500ce for a full test report.

File: easybuild/easyconfigs/s/SCons/SCons-3.0.1-foss-2017b-Python-2.7.14.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+checksums = ['24475e38d39c19683bc88054524df018fe6949d70fbd4c69e298d39a0269f173']
+
+dependencies = [('Python', '2.7.14')]
+
+sanity_check_paths = {
+    'files': ['bin/scons', 'bin/scons-time', 'bin/sconsign'],
+    'dirs': ['lib/%(namelower)s-%(version)s/%(name)s'],
+}
+
+options = {'modulename': False}
```
Comment (wpoely86) [inline easybuild/easyconfigs/s/SCons/SCons-3.0.1-foss-2017b-Python-2.7.14.eb]: this requires a comment

File: easybuild/easyconfigs/c/Cantera/Cantera-2.3.0-foss-2017b-Python-2.7.14.eb [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    '73b49e8fc73adf221b956980e5ef2a7808e6a81680f6e97ce846a159ae6d6966',  # Cantera-2.3.0_fix-test-reprod.patch
+    'e638ae44a25de9e3c38c21daa5d388f42106f9289a87f1fd01be3e985d7e7d54',  # Cantera-2.3.0_fix-recent-Cython.patch
+]
+
+dependencies = [
+    ('Python', '2.7.14'),
+    ('Boost', '1.65.1'),
+    ('SUNDIALS', '2.7.0'),
+    ('fmt', '3.0.2'),
+    ('googletest', '1.8.0'),
```
Comment (wpoely86) [inline easybuild/easyconfigs/c/Cantera/Cantera-2.3.0-foss-2017b-Python-2.7.14.eb]: build dep?

Comment (boegel) [reply easybuild/easyconfigs/c/Cantera/Cantera-2.3.0-foss-2017b-Python-2.7.14.eb]: Yeah, you're right, `googletest` only provides header files and static libraries, same for `fmt`.

File: easybuild/easyconfigs/s/SCons/SCons-3.0.1-foss-2017b-Python-2.7.14.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/s/SCons/SCons-3.0.1-foss-2017b-Python-2.7.14.eb]: fixed and made consistent via #5515

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
node2596.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/76c89e32f40c1d0cd783c5766e133f46 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
dell.lan - Linux fedora 27, Intel(R) Core(TM) i7-4600U CPU @ 2.10GHz, Python 2.7.14
See https://gist.github.com/5fc054c4752a7f01bb80138bb651b8c2 for a full test report.

**Comment by verdurin** [comment]
I'm happy to merge, @wpoely86 should probably have the final say.

**Pull request merged**